kani-driver/src/args.rs

```diff
@@ -314,9 +315,14 @@ pub struct CargoArgs {
     /// Build all packages in the workspace
     #[arg(long)]
     pub workspace: bool,
+
     /// Run Kani on the specified packages.
     #[arg(long, short, conflicts_with("workspace"), num_args(1..))]
     pub package: Vec<String>,
+
+    /// Run Kani on the specified packages.
```

Comment:
Message should be "Exclude the specified packages".

Reply:
Done.